lmfit/model.py

```diff
@@ -1497,6 +1502,9 @@ def eval_uncertainty(self, params=None, sigma=1, **kwargs):
            < 1, it is interpreted as the probability itself. That is,
            ``sigma=1`` and ``sigma=0.6827`` will give the same results,
            within precision errors.
+        3. Also sets attributes of `dely` for uncertainty of the model (same as return value)
```

Comment:
what do you mean by "same as return value" - same type (i.e., numpy.ndarray) as return value, or...?

Reply:
same value.  In the current version, the method returns an array.  That return value should not break, but it's a little `dely` for each component as a side-effect, but not set `dely` for the full thing.  So this will now do both.  We could deprecate the return value, but it does not seem necessary.